index.html

```diff
@@ -19,6 +19,7 @@
 			<nav class="content">
 				<a href="index.html"><h1 class="gittext lowercase">Git<span>for Windows</span></h1></a>
 				<ul class="list-unstyled">
+					<li><a href="governance-model.html">Governance Model</a></li>
```

Comment:
When I resized my browser window to sizes close to a typical tablet, the footer looked more pleasant than the header, but I'd prefer to keep them identical, for visual pleasure.